### PR TITLE
Feat: Search

### PR DESCRIPTION
**Pull request by pritam-kr**
- Created a new branch p1
- changed user credential
- changed username in the post 
- put a text if you don't have any post
- Small validation in creating a new post's modal.
- color change of edit and delete button of a post 
- Top bar fixed
- Added search bar 
- navigate to the profile page user after a search

### CODE REVIEW

File: comments.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- Created a new branch p1
+- changed user credential
+- changed username in the post 
+- put a text if you don't have any post
+- Small validation in create new post's modal.
+- color change of edit and delete button of a post 
+- Top bar fixed
+- Added search bar
```
Comment (varnita619) [inline comments.md]: proper description

File: src/Components/SearchResult/SearchResult.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+export { SearchResult };
+
+{
+  /* <div className="flex items-center">
+<img
+  src="https://res.cloudinary.com/dhqxln7zi/image/upload/v1652867276/istockphoto-1168369629-612x612_wko9dt.jpg"
+  alt="users-avatar"
+  className="post-avatar"
+/>
```
Comment (varnita619) [inline src/Components/SearchResult/SearchResult.jsx]: remove comments

**Review by varnita619 — APPROVED** [review]
Code is clean and modular
Everything is working as expected
Ready to merge

**Review by samyakshah3008 — APPROVED** [review]
Clean code and UI
Good for approval

**Pull request merged**